packages/design-system/src/routes/index.tsx

```
@@ -1,10 +1,11 @@
 import Sidebar from '@layouts/Sidebar';
 import ButtonExampleDocs from '@pages/ButtonExampleDocs';
+import CalendarDoc from '@pages/CalendarDoc';
+import DropdownDoc from '@pages/DropdownDoc';
 import IconDoc from '@pages/IconDoc';
 import LandingPage from '@pages/LandingPage';
-import LogoDoc from '@pages/LogoDoc';
-import CalendarDoc from '@pages/CalendarDoc';
 import LandingPage from '@pages/LandingPage';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'LandingPage'. Consider to delete it or rename it.

'LandingPage' is defined here:


(lint/suspicious/noRedeclare)